### ISSUE TITLE
Unable to quit NVDA when about dialog open.

### ISSUE THREAD

**Issue by feerrenrut** [nvaccess/nvda#6351]
Feedback on GUI, received via email.

> when the about dialog is visible, you cannot quit nvda, close other nvda windows/dialogs or do antyhing with nvda at all until you hit ok in the about dialog.  this is despite the fact that you can alt tab away from it etc.  however, if i try to quit nvda for example, which does not work, then i close the about dialog by hitting ok, my quit action is then taken.  it seems like actions are queued up and then acted upon once about is closed.

**Comment by jcsteh** [comment]
This is covered by #1740.
